packages/ui-components/package.json

```
@@ -37,15 +37,16 @@
         "test-u": "jest --ci --forceExit --colors  --silent -u"
     },
     "dependencies": {
+        "@fluentui/react": "8.66.2",
         "react-movable": "2.4.0",
         "react-virtualized": "9.22.3",
-        "sanitize-html": "2.7.1"
+        "sanitize-html": "2.7.1",
+        "uuid": "3.4.0"
     },
```

Per https://stackoverflow.com/questions/30451556/what-is-the-correct-way-of-adding-a-dependency-to-react-in-your-package-json-for we should probably add this. What do you think?

```suggestion
    },
    "peerDependencies": {
        "react": ">16.14.0",
        "react-dom": ">16.14.0"
        },
```